models/modelo.ts

```diff
@@ -0,0 +1,27 @@
+import { Tiempo } from "./tiempo.ts";
+import { Operacion } from "./operacion.ts";
```

Comment:
```suggestion
import { Tiempo } from "./tiempo";
import { Operacion } from "./operacion";
```
Los imports en typescript no deben de incluir la extensión de archivo

Reply:
La razón por la que he puesto las extensiones es porque he usado el runtime recomendado Deno y para Deno, es necesario explicitar la extensión .ts (https://github.com/denoland/deno/issues/2506). Si tienes pensado utilizar otro runtime lo cambio sin problemas.

Reply:
@vdeq79 he abierto [una issue](https://github.com/arsa-dev/OMP-Logic/issues/12) donde discuto la elección del runtime, a modo de resumen creo que es más conveniente el uso de NodeJS frente a Deno, no obstante, puedes añadir cualquier cosa que consideres oportuna. Sino, esta conversación closes #12

Reply:
Yo os animaría a que usárais deno. Pero es cuestión vuestra.

Reply:
¿@JJ has revisado la issue #12? Me gustan los conceptos sobre los que nace Deno, aunque no lo veo tan madurado como para utilizarlo en el proyecto. Encuentro poco soporte y bastantes problemas de compatibilidad de elementos muy extendidos, por ejemplo si se quiere incluir en un futuro algún orm (uno de los más extendidos es typeorm) pues nos encontramos con este otro problema de compatibilidad: https://github.com/typeorm/typeorm/issues/6123 - https://stackoverflow.com/questions/68413239/how-to-import-typeorm-in-deno

Quizás si que lo usaría para alguna solución mucho más concreta donde no se pretenda extender el producto para usar estas tecnologías que comento en la issue.

Reply:
No vais a usar ORM en este proyecto. Y con deno tienes cosas como deno deploy que pueden ser muy interesantes como PaaS.

Reply:
Sí, lo comentaba como otro ejemplo más de algo que suele haber en cualquier servicio, aunque creo que ya lo comentamos hace tiempo en clase que no íbamos a entrar en ese tema en la asignatura. Pero los otros que comento en la issue como la generación de openapi a través de metadatos en modelos y controladores, etc. Si los considero buenas prácticas para las que actualmente en deno sería posible aplicarlas pero a más bajo nivel. Además, aunque no tenga calificación en la asignatura, posteriormente me gustaría continuar el proyecto usando la misma base de código (aunque por supuesto si no es posible tengo el foco principalmente en superar objetivos).

Por cierto he estado revisando también deno deploy, y sí efectivamente se ve muy interesante aunque aún viendo como otro punto a su favor no lo veo algo decisivo porque pienso que aunque sea muy buena opción, cuando se pone una solución en manos de un tercero para alojarlo siempre debemos de tener una segunda opción lista en un plan de recuperación de desastres, por tanto también habría que evaluarlas y desarrollarlas, ya que aunque nos facilite deno el desplegar soluciones por el hecho de usar deno en sí, también debemos como mínimo conocer como hacerlo sin deno deploy.